core/templates/dev/head/pages/collection_player/collection_player.html

```diff
@@ -58,6 +71,113 @@
         left: 42px;
         position: absolute;
       }
+      @media only screen and (max-width : 600px)
+      {
+        .oppia-collection-player-title-font
+        {
+          font-size: 32px;
+        }
+      }
+      exploration-summary-tile md-card {
```

Comment:
In one of the emails Mark mentioned there is a double shadow when the preview card shows, it is because the the preview card is displayed over the blank card. So canceling box-shadow for preview card solves the double shadow problem.